docs/content/en/docs/operator-manual/piped/configuring-notifications.md

```diff
@@ -14,7 +14,7 @@ Notification configuration including:
 - a list of `Receiver`s which used to know how to send events to the external service
 
 [Notification Route](/docs/operator-manual/piped/configuration-reference/#notificationroute) matches events based on their metadata like `name`, `group`, `env`, `app`.
-Below is the list of supporting event names and their groups.
+Below is the list of supporting event names and their groups.(Events of `APPLICATION~` are not supported yet.)
```

Comment:
How about adding a new checkbox column (e.g. `Supported`) into the table below instead of this?

Reply:
Cool

Reply:
The following picture is the checkbox column. Sounds good?

![Screen Shot 2021-10-04 at 14 49 07](https://user-images.githubusercontent.com/59436572/135800536-7c571da0-162d-47e7-8b27-3254a88f1645.png)